rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -1380,3 +1393,23 @@ game_info game_list_frame::GetGameInfoFromItem(QTableWidgetItem* item)
 
 	return var.value<game_info>();
 }
+
+QColor game_list_frame::getGridCompatibilityColor(const QString string)
+{
+	static const QColor invalidColor = QColor();
+	if (!m_drawCompatStatusToGrid || m_isListLayout)
+	{
+		return invalidColor;
+	}
+	if (string.isEmpty())
+	{
+		return invalidColor;
+	}
+	return QColor(string);
+}
+
+void game_list_frame::SetCompatibilityInGrid(bool show)
+{
+	m_drawCompatStatusToGrid = show;
+	Refresh(true);
```

Comment:
Is true really needed? (Cant look right now)

Reply:
Refresh doesn't draw new icons unless it loads games from the drive. User would have to change icon size after changing the setting to force the emulator to draw icons again.

Reply:
Oh, just noticed that there's another method which repaints the icons without having to call full refresh.